setup.py

```diff
@@ -1,26 +1,30 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
+import io
+import re
 from setuptools import setup
 
+
 with open('README.rst') as readme_file:
     readme = readme_file.read()
 
 with open('HISTORY.rst') as history_file:
     history = history_file.read()
 
-requirements = [
-    'Click>=6.0',
-    # TODO: put package requirements here
-]
+with io.open("milhoja/__init__.py", "rt", encoding="utf8") as f:
+    version = re.search(r'__version__ = \'(.*?)\'', f.read()).group(1)
+
 
-test_requirements = [
-    # TODO: put package test requirements here
+install_requires = [
+    'Click>=6.0',
+    'cookiecutter>=1.6.0',
+    'six>=1.12.0',
+    # You'll also need to install libgit2 to get this to work.
```

Comment:
Any plans to containerize or use Uranium so we can create a developer experience without polluting our system dependencies?

Reply:
I've got it on the list to add Docker, wasn't going to bother with `uranium` as we're aiming open source this stuff. https://zbrt.atl.zillow.net/browse/ZOMLINFRA-407